src/lib.rs

```diff
@@ -173,16 +173,21 @@ pub struct OfferInfo {
     pub status: OfferStatus,
 }
 
+pub(crate) struct UserPreferences {
+    fiat_currency: String,
+    timezone_config: TzConfig,
+}
+
 // TODO remove dead code after breez sdk implementation
 #[allow(dead_code)]
 pub struct LightningNode {
+    user_preferences: Arc<Mutex<UserPreferences>>,
```

Comment:
Seems to be somewhat redundant with the field `data_store`

Reply:
You now have the field `user_preferences` that stores data of the type `UserPreferences` and also the field `data_store`, which stores data of the type `UserPreferences` (and `Connection`)